selfdrive/car/car_helpers.py

```diff
@@ -70,7 +71,7 @@ def fingerprint(logcan, timeout):
     # broadcast immediately
     if len(candidate_cars) == 1 and st is not None:
       # TODO: better way to decide to wait more if Toyota
-      time_fingerprint = 1.0 if ("TOYOTA" in candidate_cars[0] or "LEXUS" in candidate_cars[0]) else 0.1
+      time_fingerprint = 1.0 if ("TOYOTA" in candidate_cars[0] or "LEXUS" in candidate_cars[0] or "HYUNDAI" in candidate_cars[0]) else 0.1
```

Comment:
this is only because Toyota Camera takes a while to start sending msgs. Is this really needed for hyundai?

Reply:
Most likely not, an artefact because this was based off Toyota.
I will kill it and update it if all okay.